geometry_msgs/Vector3[] pos
std_msgs/Float32[] r
